nirc_ehr/test/src/org.labkey.test/tests.nirc_ehr/NIRC_EHRTest.java

```diff
@@ -76,6 +76,7 @@ public class NIRC_EHRTest extends AbstractGenericEHRTest implements PostgresOnly
     private static String deadAnimalId = "D5454";
     private static String departedAnimalId = "H6767";
     private static String aliveAnimalId = "A4545";
+    private static String orchardFileLocation;
```

Comment:
Since we know the file location, there's no need to delay setting the value (will need to create it later though)
```suggestion
    private static final String orchardFileLocation = TestFileUtils.getTestTempDir();
```